### PR TITLE
Fixing dependecies

### PR DESCRIPTION
**Pull request by erikbosch**
Dependabot cannot fix them as NOTICE file needs manual update.
Based on PRs #280, #281, #283, #288, #289 and

```
sudo npm install
sudo npm audit fix
```

Note the big change in the NOTICE file - actually the license check fails on main, see https://github.com/eclipse-velocitas/cli/actions/workflows/check-licenses.yml

### CODE REVIEW

File: NOTICE-3RD-PARTY-CONTENT.md [hunk 1]
```diff
@@ -3,34 +3,486 @@
 ## JavaScript
 | Dependency | Version | License |
 |:-----------|:-------:|--------:|
+|@aashutoshrathi/word-wrap|1.2.6|MIT|
```
Comment (erikbosch) [inline NOTICE-3RD-PARTY-CONTENT.md:6]: @wba2hi @dennismeister93 

Do you know if this big change in license checker output is expected?

This seems to be related to an update in license checker itself, as also a build on main fails, see https://github.com/eclipse-velocitas/cli/actions/runs/9758056133

Comment (dennismeister93) [reply NOTICE-3RD-PARTY-CONTENT.md:6]: it seems that the license checker now adds all dependencies which are needed by our dependencies (all which are downloaded and inside the node_modules) and not only those we specified in the package.json.

That justifies the big change.
But i dont know if only our defined dependencies inside package.json are necessary or if we should explicitly list all of them

Comment (wba2hi) [reply NOTICE-3RD-PARTY-CONTENT.md:6]: License Checker was basically updated from 7.1.0 to version 7.2.1. If this is the cause for the imense number of new entries is hard to tell. 

Could be related the change "[Add more license detection](https://github.com/pivotal/LicenseFinder/commit/b3dde46e2018aef9b6436a04777ec33d40a134e4)" in version [7.2.0](https://github.com/pivotal/LicenseFinder/releases/tag/v7.2.0)

Unfortunately I am not deep enough into JS to know how to verify the dependency and transitive dependencies tree.

Comment (dennismeister93) [reply NOTICE-3RD-PARTY-CONTENT.md:6]: The change you mentioned just added new licenses.
The big change of the licenses found is the difference of:
"npm list" (lists explicitly needed dependencies for the project) and "npm list --all" (lists all transitive)

Comment (dennismeister93) [reply NOTICE-3RD-PARTY-CONTENT.md:6]: in the end i would say, its better having them listed than not

**Review by dennismeister93 — APPROVED** [review]
lgtm

**Pull request merged**